starter-code/app.js

```diff
@@ -8,6 +8,13 @@ const hbs          = require('hbs');
 const mongoose     = require('mongoose');
 const logger       = require('morgan');
 const path         = require('path');
+const session      = require('express-session');
+const Mongostore   = require('connect-mongo')(session)
+const flash        = require('connect-flash');
+const passport     = require('passport');
+const LocalStrategy = require('passport-local').Strategy
+
+const bcrypt = require('bcrypt');
 
```

Comment:
You forgot to add/npm install express-session, connect-mongo, connect-flash, passport, passport-local and bcrypt in your package.json. Make sure you have all your project dependencies in future projects